src/azure-cli/azure/cli/command_modules/sql/custom.py

```diff
@@ -462,6 +462,30 @@ class ComputeModelType(str, Enum):
     serverless = "Serverless"
 
 
+class DatabaseEdition(str, Enum):
```

Comment:
May I know if we have DatabaseEdition in python SDK?

Reply:
Not anymore. It was removed as enum in latest SDK package. Thats why I added it as enum inside custom.py, since code still uses some of its values